habitat/plan.sh

```diff
@@ -0,0 +1,208 @@
+pkg_name=chef-workstation
+pkg_origin=chef
+pkg_maintainer="The Chef Maintainers <humans@chef.io>"
+pkg_description="Chef Workstation - Opinionated tools for getting the most out of the Chef ecosystem"
+pkg_license=('Apache-2.0')
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
+ruby_pkg="core/ruby26"
+pkg_build_deps=(
+  core/make
+  core/gcc
+  core/go
+  core/gcc-libs
+  core/pkg-config
+  # We make this a build dependency since we are going to import
+  # the generated binary into our list of binaries.
+  # @afiune: Not sure if this is the right pattern but it works.
+  chef/chef-analyze
```

Comment:
Does the generated binary not get pulled into the list of bins if we put this in pkg_deps?

Reply:
From the hab docs, `pkg_build_deps` is " An array of the package dependencies needed only at build time. ". It seems we'll want chef-analyze at runtime too, so will it still work correctly to include this in `pkg_deps` instead?